src/kernel/checks.cpp

```diff
@@ -24,7 +24,7 @@ bool Clang_IndVarSimplify_Bug_SanityCheck() {
         }
         last = *it;
     }
-    return false;
+    return true;
```

Comment:
This breaks the sanity check. If the test is failing for you, you should get a compiler that doesn't have the bug. It could very well impact other code!